### DISCUSSION TITLE
Adopt Superior Scraper Called "where to buy 3080", doesn't require web browser, is much better

### DISCUSSION THREAD

**Discussion by rugabunda** [jef/streetmerchant#1640] [Ideas and feature requests]
Please adopt the code from this 3080 script scraper and adopt it to your own preference, it is far superior, much quicker, and uses only 40-120 kilobytes per second of data transfer to update every single cards stock status literally every 10 seconds or so. Even then, it does not require a web browser, and uses almost no cpu power, but will open a browser when stock is detected. 
https://github.com/LesterLyu/where-to-buy-3080

![image](https://user-images.githubusercontent.com/38260267/104086967-32ec2780-5254-11eb-94c3-6c3583fcb166.png)

**Comment by lec6213** [comment]
Looks interdating but it appears to support Canada only and if it does not you should significantly more clear about that  and from the title and the screen shot is appears to be setup for 3080 only?  Is it expandable and will it be supported going forward with more stores and models.

**Comment by wofnull** [comment]
The script itself was not updated since october and lacks a base config for most stores , it would need much time to put it to work with all stores and make it overall configurable for most users / locations.

but i think if jef would take a look, eventually in an upcoming update ( 5.0  or higher ) it would be possible to get rid of the overall bad chrome performance.
